src/cli/injected/recorder.webpack.config.js

```diff
@@ -20,6 +20,7 @@ const InlineSource = require('../../server/injected/webpack-inline-source-plugin
 module.exports = {
   entry: path.join(__dirname, 'recorder.ts'),
   devtool: 'source-map',
+  mode: 'development',
```

Comment:
Dev mode react is pretty different from prod mode. I'll look into us getting a production webpack build step

Reply:
maybe we can make it based on the `NODE_ENV` environment variable like
```js
process.env.NODE_ENV === 'production' ? 'production' : 'development'
```

I think thats common and I saw this already a few times.

Reply:
So, we used to have everything in 'development' mode. Now, I want to minify the trace viewer, so we use 'production' mode in build.js. However, turning everything into 'production' does not work for our injected scripts, so I manually set 'development' in most webpack configs. Does that make sense?

Reply:
Curious why it does not work for the injected scripts?

Reply:
Because we expect something like `.default` being exported from the result of webpack bundle. I am sure this can be solved, but nobody looked into it, because benefits are not clear.